docs/markdown/Creating-releases.md

```diff
@@ -60,6 +51,21 @@ for example when done in CI that already does its own testing.
 So with `--no-tests` you can tell Meson "Do not build and test generated
 packages.".
 
+## Modifying release archive contents
+
+By default `meson dist` includes all source code under version control. In
+order to exclude tracked files, include uncommitted files, or modify existing
+files, dist scripts can be used with `meson.add_dist_script`. For example:
+
+```
+meson.add_dist_script('gen_somefile.py')
+```
+
+Dist scripts work for any supported VCS. In addition, if Git is used as the
```

Comment:
Note that this is not yet true (see https://github.com/mesonbuild/meson/issues/12829#issuecomment-1935373402). However, this should indeed be the case because the dist script mechanism is orthogonal to the specific VCS program.

Reply:
It will be true when gh-12840 lands, right? I left the text unchanged, counting on that. If not, I'm not sure what to change it too - there's also some amount of Subversion and Bazaar support, and I'm not sure about the status there. I imagine at least simple dist scripts (like `rm a-file`) work?